config/interpolate_walk.go

```diff
@@ -54,6 +54,9 @@ type interpolationWalkerContextFunc func(reflectwalk.Location, ast.Node)
 
 func (w *interpolationWalker) Enter(loc reflectwalk.Location) error {
 	w.loc = loc
+	if loc == reflectwalk.WalkLoc {
```

Comment:
Correct - this is in lieu of a constructor to set the default value for the interpolate walker struct